### ISSUE TITLE
owner attribute

### ISSUE THREAD

**Issue by videlec** [robertwb/sage-patchbot#80]
When launching the patchbot we got
```
Traceback (most recent call last):
  File "/home/vincent/sage_patchbot/local/bin/patchbot/patchbot.py", line 1280, in report_ticket
    report['owner'] = self.owner
AttributeError: 'Patchbot' object has no attribute 'owner'
```

**Comment by fchapoton** [comment]
solved in patchbot 2.5.6, that should arrive soon (no trac ticket yet)

**Comment by fchapoton** [comment]
needs review at trac ticket #20528

**Comment by fchapoton** [comment]
This is solved in patchbot 2.5.7, please review #20528.
